### PR TITLE
VET TEC: Update OMB control numbers for VET TEC on VA.gov #17467

### PR DESCRIPTION
**Pull request by zurbergram**
## Description

As a VET TEC applicant on VA.gov, I want to know the OMB respondent burden, control number, expiration date and privacy implications so that I can understand how difficult the form will be to complete.

## Testing done
- local
- unit

## Screenshots
Screen Shot 2019-04-26 at 3 59 56 PM
Screen Shot 2019-04-26 at 3 54 49 PM

## Acceptance criteria
- [ ] The following OMB values are displayed on the Introduction Page:
   - Respondent Burden: **10 minutes**
   - OMB Control #: **2900-0866**
   - Expiration Date: **04/30/2022**
- [ ] The text displayed when clicking the "Privacy Act Statement" link is replaced with the mentioned in AC#2 of https://github.com/department-of-veterans-affairs/vets.gov-team/issues/17467

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by devin-mccurdy — APPROVED** [review]
LGTM

**Comment by zurbergram** [comment]
@department-of-veterans-affairs/frontend-review-group Could I get a code review on this at your earliest convenience. Thank you!

File: src/applications/edu-benefits/0994/components/OMBInfo.jsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
+import PropTypes from 'prop-types';
```
Comment (jbalboni) [inline src/applications/edu-benefits/0994/components/OMBInfo.jsx]: Is this component different than https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/ombinfo/?

Comment (zurbergram) [reply src/applications/edu-benefits/0994/components/OMBInfo.jsx]: yes, 
- “respondent burden” removed “(Office of Management and
     Budget) ”
- “privacy act notice” was larger change
additionally see https://dsva.slack.com/archives/CG6N59X40/p1556311972017800

**Review by rianfowler — APPROVED** [review]

**Pull request merged**